### DISCUSSION TITLE
Make placeholder text and field limit overridable

### DISCUSSION THREAD

**Discussion by ryanleichty** [craftcms/cms#15660] [Ideas] [answered]
It would be great if you could override the _placeholder text_ and _field limit_ of fields. Currently, this is one of the criteria for us when deciding if we need to create a new field versus using a generic field that we can customize the name and handle of.

This would be really powerful combined with variables (https://github.com/craftcms/cms/discussions/15635).

![CleanShot 2024-09-04 at 09 12 05@2x](https://github.com/user-attachments/assets/8c908756-64a8-4eef-96fc-5e42929f8de2)

**Comment by ryanleichty** [comment]
There's a little bit of overlap with this discussion (https://github.com/craftcms/cms/discussions/14566). In general, it seems that making more items overridable would be helpful.

**Comment by brandonkelly** [comment, marked as answer]
This is basically a duplicate of #8688. See [this comment](https://github.com/craftcms/cms/discussions/8688#discussioncomment-2811602) on why it’s a little challenging. Still something that could be considered down the road, but #14566 is a more practical solution.

**Reply by ryanleichty** [reply]
Yeah, I saw the _clone field_ discussion which would be helpful, but it's not a complete solution for what we're looking for. I kind of assumed there was more complexity involved than what might be expected 👍
